client/landing/stepper/declarative-flow/tailored-ecommerce-flow.ts

```diff
@@ -22,6 +22,14 @@ import { USER_STORE, ONBOARD_STORE, SITE_STORE } from '../stores';
 import getQuantityFromStorageType from '../utils/get-quantity-from-storage-slug';
 import { getLoginUrl } from '../utils/path';
 import { recordSubmitStep } from './internals/analytics/record-submit-step';
+import CheckPlan from './internals/steps-repository/check-plan';
+import CreateSite from './internals/steps-repository/create-site';
+import DesignCarousel from './internals/steps-repository/design-carousel';
+import DomainsStep from './internals/steps-repository/domains';
+import ProcessingStep from './internals/steps-repository/processing-step';
+import StoreProfiler from './internals/steps-repository/store-profiler';
+import WaitForAtomic from './internals/steps-repository/wait-for-atomic';
+import WaitForPluginInstall from './internals/steps-repository/wait-for-plugin-install';
```

Comment:
This looks like it may have been from a poor rebase/merge -- I think these imports are no longer used due to us loading the steps asynchronously.
```suggestion
```